tests/integration/std/test_collections.py

```diff
@@ -21,3 +21,20 @@ def main() -> int:
     run_int_fn(compiled, sum((i + 1) * x for i, x in enumerate(reversed(list(range(10))))))
 
 
+def test_priority_queue(validate, run_int_fn) -> None:
```

Comment:
This test fails on execution because the sum isn't quite correct - it should work with the priority set to `i` instead of `9 - i`.

Reply:
I definitely did the math wrong! I'm leaving the priority opposite the value to ensure elements come out of the queue in the  correct order, but I fixed the math. Ironically, it looks even more similar to the stack test, but I added some comments explaining why it's put together the way it is.